.travis.yml

```diff
@@ -0,0 +1,47 @@
+sudo: required
+language: go
+go:
+  - 1.7.x
+addons:
+  apt:
+    packages:
+    - git
+    - make
+    - curl
+
+env:
+  - APP=openvpn-api
+
+install:
+- make init
+- make go:deps-build
+- make go:deps-dev
+
+script:
+- make go:deps
+- make go:test
+- make go:lint
+- make go:build-all
+
+deploy:
+  provider: releases
+  api_key: $GITHUB_OAUTH_TOKEN
+  file:
+    - release/$APP_darwin_386
```

Comment:
What do you think about using wildcards here? https://github.com/cloudposse/sudosh/blob/master/.travis.yml#L27